src/it/warResources/src/main/webResource-filtering/help-globalConfig.html

```diff
@@ -7,12 +7,12 @@
   </p>
   <p>
     You can also filter using interpolation like: 
-    \${prop1\} ${prop1}
+    \${prop1} ${prop1}
     or 
     \@prop2\@ @prop2@
     and these will be replaced at build time
     it is even possible to use system properties
-    \@java.class.version\@ ${java.class.version}
+    \@java.vendor.url\@ ${java.vendor.url}
```

Comment:
class version is of the JVM not the compiled classes - so this would on java17 etc.  replaced with vendor.rul which is very jvm specific - but it is 2023 so if this is not a https:// url then someone should fix the JVM...